tardis/montecarlo/src/cmontecarlo.c

```diff
@@ -7,8 +7,16 @@
 #include "abbrev.h"
 #include "cmontecarlo.h"
 #include "io.h"
-
-
+long long int count_resonance = 0;
```

Comment:
@SooryaN - as a first step to completing the task, defining a global resonance counter is completely fine. However, keep in mind that eventually the goal is to count resonances on a per-line basis.